docs/usage/topology_aware_routing.md

```diff
@@ -75,9 +75,31 @@ The `gardener-resource-manager` Service that is part of the Shoot control plane
 
 The `vpa-webhook` Service that is part of the Shoot control plane is topology-aware. It is consumed by the kube-apiserver for the webhook communication.
 
-## How to enable the topology-aware routing?
+## Topology-aware Services in the garden runtime cluster
 
-The topology-aware routing functionality can be enabled in the Seed specification:
+##### virtual-garden-etcd-main-client and virtual-garden-etcd-events-client
+
+The `virtual-garden-etcd-main-client` and `virtual-garden-etcd-events-client` Services are topology-aware. `virtual-garden-etcd-main-client` is consumed by `virtual-garden-kube-apiserver` and `gardener-apiserver`, `virtual-garden-etcd-events-client` is consumed by `virtual-garden-kube-apiserver`.
+
+##### virtual-garden-kube-apiserver
+
+The `virtual-garden-kube-apiserver` Service is topology-aware. It is consumed by `virtual-garden-kube-controller-manager`, `gardener-controller-manager`, `gardener-scheduler`, `gardener-admission-controller`, extension admission components, `gardener-dashboard` and other components.
+
+> Note: Unlike the other Services, the `virtual-garden-kube-apiserver` Service is of type LoadBalancer. In-cluster components consuming the `virtual-garden-kube-apiserver` Service by its Service name will have benefit from the topology-aware routing. However, the TopologyAwareHints feature cannot help with external traffic routed to load balancer's address - such traffic won't be routed in a topology-aware manner and will be routed according to the cloud-provider specific implementation.
+
+##### gardener-apiserver
+
+The `gardener-apiserver` Service is topology-aware. It is consumed by `virtual-garden-kube-apiserver`. The aggregation layer in `virtual-garden-kube-apiserver` proxies requests sent for the Gardener API types to the `gardener-apiserver`.
+Although the component is not yet managed by gardener-operator, its helm chart allows enabling the topology-aware routing.
```

Comment:
Perhaps drop this line and also L98. It's easy to miss, and soon this will change.